aten/src/ATen/Context.cpp

```diff
@@ -335,6 +335,9 @@ at::BlasBackend Context::blasPreferredBackend() {
     static const bool hipblaslt_preferred = []() {
       static const std::vector<std::string> archs = {
           "gfx90a", "gfx942",
+#if ROCM_VERSION >= 60300
```

Comment:
@fjankovi Is hipblasLT more performant that hipblas even in ROCm6.3?
cc @jeffdaily